sys/dev/netmap/netmap.c

```diff
@@ -513,7 +511,9 @@ int netmap_generic_mit = 100*1000;
  * Anyway users looking for the best performance should
  * use native adapters.
  */
+#ifdef linux
```

Comment:
I took the liberty to add the #ifdef here as well, if you think that is unnecessary I can also remove it

Reply:
this looks good!